package-scope.qmd

```diff
@@ -1,55 +1,43 @@
 ---
-title: "Package scope"
+title: "Plotting functionality"
 ---
 
-## Visualisation in R packages
-
-The use of visualisation is a fundamental step in scientific analysis and a [core component of data science](https://r4ds.had.co.nz/introduction.html). However, the question of whether a package should include plotting functionality is not obvious.
-
-::: {.callout-note title="Not all package plotting requires are equal"}
-
-This section details the Epiverse-TRACE reasoning for the inclusion or exclusion of functionality around plotting. It is not a commentary generalisation to all R packages.
+Visualisation is key to (data) science and in this section we detail how we approach plotting functionality for our Epiverse packages.
 
+:::{.callout-warning}
+There is ongoing work around theming of plots. Please see [{epiversetheme}](https://epiverse-trace.github.io/epiversetheme/) for more information on this.
 :::
 
-## Epiverse-TRACE philosophy
-
-The Epiverse-TRACE philosophy is to only include minimal, dependency-free, plotting functionality, and to instead provide examples and templates of plotting using {ggplot2} in vignettes. An example of the inclusion of a plotting function is the [`<epidist>` plotting method](https://epiverse-trace.github.io/epiparameter/reference/plot.epidist.html) in {epiparameter}.
+## Approach
 
-Epiverse-TRACE packages aim to provide comprehensive documentation that explains exported functions and demonstrates their applications, both through [{roxygen2}](https://roxygen2.r-lib.org/) function-specific documentation, or via longer-form vignettes. It is in the vignettes that Epiverse-TRACE packages show examples of how the data or model outputs can be visualised, with the associated code chunks required to produce the plot.
+Our approach to plotting functionality is twofold:
 
-It is with this provision of plotting code chunks we hope to provide the community with a resource from which their desired plots can be produced. Vignettes use the {ggplot2} package to provide easily extensible plots with the addition of [layers](https://rpubs.com/hadley/ggplot2-layers).
+1. We minimize dependencies used for plotting functions in our packages.
 
-Epiverse-TRACE packages are not developed uniformly and certain packages may deviate from this philosophy. In these cases the reasoning for the difference should be documented in that package's design vignette.
+2. We provide detailed plotting examples throughout our vignettes, which may rely on external dependencies and include reproducible code. 
```

Comment:
All (or a least the vast majority) of vignette plotting only requires {ggplot2}. It might be better to clarify this here as external dependencies may be interpreted as any other plotting package is allowed. Anything used in the vignettes will need to be stated as a suggested dependency, and a lot of package installation in R installs suggested dependencies by default, so it can quickly make a package not so lightweight.

Reply:
Good point - tried clarifying this by naming them as "common external dependencies" for now.